src/sage/graphs/convexity_properties.pyx

```diff
@@ -674,3 +671,155 @@ def geodetic_closure(G, S):
     free_short_digraph(sd)
 
     return ret
+
+
+def is_geodetic(G):
+    r"""
+    Check whether the input (di)graph is geodetic.
+
+    A graph `G` is *geodetic* if there exists only one shortest path between
+    every pair of its vertices. This can be checked in time `O(nm)` in
+    unweighted (di)graphs. Examples of geodetic graphs are trees, cliques and
+    odd cycles. See the :wikipedia:`Geodetic_graph` for more details.
+
+    (Di)graphs with multiple edges are not considered geodetic.
+
+    INPUT:
+
+    - ``G`` -- a Sage graph or digraph
+
```

Comment:
Do we need "Sage" here?

Reply:
I removed the term Sage. It's sometimes used in graphs (not unified).